### DISCUSSION TITLE
Service Referer Question

### DISCUSSION THREAD

**Discussion by runneals** [Esri/SystemLogParser#11] [Q&A]
When looking at the Service Referer sheet for IIS, I'm noticing a lot of esri.com/AGO/[GUID] and esri.com/[GUID]. Where do these come from and is there a way to identify an application? Thanks! @AaronLopez5283

**Comment by AaronPLopez** [comment]
Hi @runneals, 
I was not sure myself, but after discussing with colleagues, that referer style is the convention that ArcGIS Pro uses.
So, if you consume services from your Site in ArcGIS Pro, you should see a similar signature.

Hope that helps.
Aaron
